fe/fe-core/src/main/java/org/apache/doris/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -93,6 +95,7 @@ public class KafkaRoutineLoadJob extends RoutineLoadJob {
     // kafka properties ，property prefix will be mapped to kafka custom parameters, which can be extended in the future
     private Map<String, String> customProperties = Maps.newHashMap();
     private Map<String, String> convertedCustomProperties = Maps.newHashMap();
+    private Map<String, Map<String, String>> convertedRackCustomProperties = Maps.newHashMap();
```

Comment:
why do you use a new property "convertedRackCustomProperties" to store rack info rather than in convertedCustomProperties?

Reply:
a new parameter **ConsumerConfig.CLIENT_RACK_CONFIG**(aka client.rack, this is a standard KafkaClient consumer config to specify the rack of the KafkaConsumer)  introduced into convertedCustomProperties. so the value of each BE may not the same.

Reply:
Adding the property convertedRackCustomimProperties makes the code look complex. I think you can set this parameter in `createRoutineLoadTask` function, just set add the parameter to `tKafkaLoadInfo.Properties`.

Reply:
@flashmouse sorry for delay, this PR doesn't seem to have any major issues, except for the need to modify it.